### PR TITLE
chore: add back authentication md

### PR DESCRIPTION
**Pull request by shuiyisong**
`authentication.md` is missing after #299. This pr mainly adds it back and put it under `User Guide`. Please take a look @nicecui

### CODE REVIEW

File: docs/user-guide/authentication.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+./greptime standalone start --user-provider=static_user_provider:cmd:<username>=<password>
+```
+
+Replace `username` and `password` with your configured username and password, and it's done!
+
+Single user configuartion is easy to setup and test, but it's not recommanded for any serious situations, for example, a production environment.
+
+### Multi-user file configuration
+
+Often, one fixed user isn't really much of a help. GreptimeDB also supports passing in a file and loads all users listed within the file.
```
Comment (xtang) [inline docs/user-guide/authentication.md:47]:
```suggestion
Frequently, having only one specific user isn't particularly useful. GreptimeDB also allows for the input of a file which then loads all users listed within it.
```

File: docs/user-guide/authentication.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+```shell
+./greptime standalone start --user-provider=static_user_provider:file:<path_to_file>
+```
+
+GreptimeDB reads the user and password on each line using `=` as a separator, just like a command-line config. For example:
```
Comment (xtang) [inline docs/user-guide/authentication.md:53]:
```suggestion
GreptimeDB reads the username and password on each line using `=` as a separator, just like a command-line config. For example:
```

**Review by xtang — APPROVED** [review]
LGTM

**Comment by shuiyisong** [comment]
Close as authentication is located [here](https://docs.greptime.com/user-guide/clients#authentication)

**Pull request closed without merging**